src/main.scss

```diff
@@ -1,62 +1,138 @@
-// >>>>>>>>>>> header <<<<<<<<<<
-header {
+// >>>>>>>>>>> global <<<<<<<<
 
-}
-.image-lukasz {
-  background-image: url("./gfx/lukasz.png");
-  height: 857px;
-  background-repeat: no-repeat;
-  background-color: #c0d3df;
-  background-position: center;
+hr {
+  border-top: 1px solid rgba(255, 255, 255, 0.9);
 }
 
-.left-box {
-  background-color: #114c7f;
-  color: white;
-  border-color: #f2da64;
-  outline: 2px solid #fff;
-  outline-offset: -20px;
+button {
+  padding: 0;
+  border: none;
+  background: none;
 }
 
-.inside-text {
-  padding: 30px;
+$bg-color-blue: #114c7f;
+$bg-color-light-blue: #c0d3df;
+$bg-color-pink: #bd3c8a;
+// >>>>>>>>>>> global end <<<<<<<<<<
+
+// >>>>>>>>>>> header <<<<<<<<<<
+.section-about-me {
+  .nav-bar {
+    ul {
+      li {
+        color: $bg-color-blue;
+        font-size: 16px;
+        font-weight: 700;
+      }
+    }
+  }
 }
 
-.row .left-box .title-box {
-  max-width: 250px;
-  margin: 0 auto;
+.section-about-me.background-image {
+  background-image: url("./gfx/lukasz.png");
+  height: 857px;
+  background-repeat: no-repeat;
+  background-color: $bg-color-light-blue;
+  background-position: center;
 }
 
-.left-box .title-box .title {
-  background-color: #BD3C8A;
-  text-align: center;
-  // max-width: 250px;
-  padding: 20px;
-  margin-top: -30px;
+.section-about-me {
+  .left-box {
+    background-color: #114c7f;
+    color: white;
+    border-color: #f2da64;
+    outline: 1px solid #fff;
+    outline-offset: -35px;
+    .inside-text {
+      font-size: 22px;
+      padding: 40px 50px 70px 50px;
+      line-height: 28px;
+      h3 {
+        font-size: 29px;
+        font-weight: 700;
+        &:nth-last-of-type(1) {
+          margin-top: 30px;
+        }
+      }
+      p {
+        margin-bottom: 30px;
+      }
+      ul {
+        font-weight: 300;
+        font-family: "Roboto Slab", serif;
+        list-style-type: none;
+        line-height: 28px;
+      }
+    }
+    .title-box {
+      max-width: 200px;
+      margin: 0 auto;
+      .title {
+        background-color: $bg-color-pink;
+        text-align: center;
+        padding: 15px;
+        margin-top: -25px;
+        font-size: 16px;
+        font-weight: 700;
+      }
+    }
+  }
 }
 
 .js {
+  font-family: "Roboto Slab", serif;
   background-color: #f2da64;
-  color: #2E2F32;
-  font-size: 40px;
-  height: 70px;
-  width: 70px;
+  color: #2e2f32;
+  font-size: 55px;
+  height: 90px;
+  width: 90px;
+  margin: -45px;
+  font-weight: 700;
 }
 
-
-.right-box {
-  background-color: #BD3C8A;
-  height: 200px;
-  outline: 2px solid #fff;
-  outline-offset: -20px;
+.section-about-me {
+  .right-box {
+    background-color: #bd3c8a;
+    height: 200px;
+    outline: 1px solid #fff;
+    outline-offset: -10px;
+    color: white;
+    margin-top: -70px;
+    .react-logo {
+      margin-left: 200px;
+      margin-top: -132px;
+    }
+    .title-box {
+      h6 {
```

Comment:
czy wszystkie h6 w twoim pliku beda mialy defaultowo 16px? jesli tak to warto to do globalnych dodac :)

Reply:
to samo z innymi tytualami, wydaje mi się, że mniej więcej starałam się utrzymać taką samą wielkość tytułów powtarzających się :)